### ISSUE TITLE
Is it possible to attach an image while editing issue?

### ISSUE THREAD

**Issue by dilawar** [stephencelis/ghi#329]
Can I attach an image to issue from my editor, say, by using markdown syntax e.g. I write the following in my editor 

    This figure ![](./myfig.png) shows the behavior after fixing #29 .

And ghi uploads the image onto git server and create appropriate link in issue body?

**Comment by lsloan** [comment]
Just to clarify: Where in the GitHub repository would it upload (commit) the image file?  Would it go into the root directory of the repo?

**Comment by dilawar** [comment]
Not really. The default server to which browser send the uploaded image is fine e.g. following image is send to cloud.githubusercontent.com/assets/id/hash_of_file.jpeg and a link was inserted in this ticket `![46fd8de5ab55378af363865f901bbd5a](https://cloud.githubusercontent.com/assets/895681/23009782/f6e49038-f43d-11e6-9189-df894a3c28a6.jpeg)` 

![46fd8de5ab55378af363865f901bbd5a](https://cloud.githubusercontent.com/assets/895681/23009782/f6e49038-f43d-11e6-9189-df894a3c28a6.jpeg)

Something similar would be nice.
